### ISSUE TITLE
Sketchfab形式のライセンス情報に対応してほしい

### ISSUE THREAD

**Issue by vyv03354** [ousttrue/UniGLTF#61]
Sketchfabからダウンロードしたモデルは、`asset.extras`に以下のようなメタ情報が自動的に付加されます。
```
 "asset": {
    "extras": {
      "author": "DWANGO Co., Ltd. (http://3d.nicovideo.jp/alicia/)",
      "license": "Other (http://3d.nicovideo.jp/alicia/rule.html)",
      "source": "http://3d.nicovideo.jp/works/td14712",
      "title": "Alicia Solid"
    }
  },
```
Sketchfab以外のglTFビューアーもこの情報に対応している場合があります。たとえば、[three glTF Viewer](https://gltf-viewer.donmccurdy.com/)はこの情報が存在する場合、作者名などを画面に表示します。

参考までに、VRMのメタ情報との対応は以下のようになります。

<table><thead>
<tr><th>asset.extras</th><th>extensions.VRM.meta</th><tr>
</thead><tbody>
<tr><td rowspan=2>author</td><td>author</td>
<tr><td>contactInformation</td>
<tr><td rowspan=2>license</td><td>licenseName</td>
<tr><td>otherLicenseUrl</td>
<tr><td>source</td><td>(なし)</td>
<tr><td>title</td><td>title</td>
</tbody></table>

authorとlicenseにURLがある場合、カッコ書きで後ろに付加します。ライセンス名は[SPDXライセンスID](https://spdx.org/licenses/)が定義されている場合、そのIDを記載します。VRMで定義されているライセンスとの対応は以下のとおりです。

|VRM|SPDX|
|:---|:---|
|Redistribution_Prohibited|(なし)|
|CC0|CC0-1.0|
|CC_BY|CC-BY-4.0|
|CC_BY_NC|CC-BY-NC-4.0|
|CC_BY_SA|CC-BY-SA-4.0|
|CC_BY_NC_SA|CC-BY-NC-SA-4.0|
|CC_BY_ND|CC-BY-ND-4.0|
|CC_BY_NC_ND|CC-BY-NC-ND-4.0|
|Other|(なし)|

VRMのCC0とCCにはバージョンが明記されていませんが、[VRM仕様のページ](https://dwango.github.io/vrm/vrm_spec/#%E3%83%A9%E3%82%A4%E3%82%BB%E3%83%B3%E3%82%B9%E3%82%BF%E3%82%A4%E3%83%97-json-extensions-vrm-meta-licensename)のリンク先からバージョンを判断しました(CCはバージョンが異なれば異なるライセンスなので、バージョンがないと意味をなしません)。

**Comment by ousttrue** [comment]
仕様が書いてあるところがわかれば、それに沿って入れたいです。

この辺でしょうか。
https://github.com/KhronosGroup/glTF/issues/839

CCのバージョンはVRMで参考にさせていただきたいと思います。

**Comment by vyv03354** [comment]
はい、そのissueと [こちらの投稿](https://github.com/KhronosGroup/glTF/issues/1051#issuecomment-424495845) を参考にしました。
